templates/init/functions/javascript/package.nolint.json

```diff
@@ -9,11 +9,11 @@
     "logs": "firebase functions:log"
   },
   "engines": {
-    "node" : "8"
+    "node": "8"
   },
   "dependencies": {
-    "firebase-admin": "~7.0.0",
-    "firebase-functions": "^2.3.0"
+    "firebase-admin": "^8.0.0",
+    "firebase-functions": "^3.0.0"
```

Comment:
Just another reminder that we have to release functions SDK before we release the CLI.